pkg/virt-controller/services/template.go

```diff
@@ -130,11 +130,11 @@ const (
 const customSELinuxType = "virt_launcher.process"
 
 type TemplateService interface {
-	RenderMigrationManifest(vmi *v1.VirtualMachineInstance, sourcePod *k8sv1.Pod) (*k8sv1.Pod, error)
-	RenderLaunchManifest(vmi *v1.VirtualMachineInstance) (*k8sv1.Pod, error)
+	RenderMigrationManifest(vmi *v1.VirtualMachineInstance, sourcePod *k8sv1.Pod, hasOwnerVm bool) (*k8sv1.Pod, error)
+	RenderLaunchManifest(vmi *v1.VirtualMachineInstance, hasOwnerVm bool) (*k8sv1.Pod, error)
```

Comment:
general note and also about this one specifically it will be outdated soon, now that i rework to support standalone VMI
so lets go back to the leftovers once it is done please

Reply:
was removed anyhow because wasn't needed anymore with the new standalone vmi support